src/libstd/sys/unix/fs.rs

```diff
@@ -149,22 +155,33 @@ impl FileAttr {
 #[cfg(not(any(target_os = "ios", target_os = "macos", target_os = "netbsd")))]
 impl FileAttr {
     pub fn modified(&self) -> io::Result<SystemTime> {
-        Ok(SystemTime::from(libc::timespec {
-            tv_sec: self.stat.st_mtime as libc::time_t,
-            tv_nsec: self.stat.st_mtime_nsec as libc::c_long,
-        }))
+        #[cfg(target_os = "emscripten")]
+        { return Ok(SystemTime::from( self.stat.st_mtim )); }
+        #[cfg(not(target_os = "emscripten"))]
+        {
+            return Ok(SystemTime::from(libc::timespec {
+                tv_sec: self.stat.st_mtime as libc::time_t,
+                tv_nsec: self.stat.st_mtime_nsec as libc::c_long,
+            }));
+        }
     }
 
     pub fn accessed(&self) -> io::Result<SystemTime> {
-        Ok(SystemTime::from(libc::timespec {
-            tv_sec: self.stat.st_atime as libc::time_t,
```

Comment:
I've had a look at doing that, but I don't think it is possible.
`st_atime()` and `st_atime_sec()` are functions available on structs that implement libstd::os::emscripted::fs::MetedataExt or libstd::sys::unix::ext::fs::MetadataExt but I believe the FileAttr struct doesn't implement either of those traits.
In this case the FileAttr struct only has access to its own inner `stat: stat64` struct.